rovilay/phone_number-generator
# Get phone numbers

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?    
    * create endpoint to fetch phone numbers
    * create endpoint to get min and max phone numbers

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/phone_number-generator.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ft-get-phone-numbers`
- set up the `.env` with the `.env.sample` file
- install dependencies - `yarn install`
- start the app - `yarn run server`
- navigate to `localhost:port/api/v1/phonenumbers` on the browser
- you should see a list of phone numbers

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
